views/admin/distributions/distributions.ejs

```diff
@@ -33,10 +33,10 @@ function deleteDistribution(id) {
     $.ajax({
       url: "/distribution/"+id,
       type: "DELETE",
-      dataType: "text",
+      dataType: "json",
     })
         .done(function(response) {
-            console.log(response);
+            console.log("Distribution deleted successfully!");
```

Comment:
If you have the info, it would be nice to include distribution ID and name in the log, since "some distribution was deleted" in the logs is less helpful.